src/legacy/core_plugins/metrics/common/annotations.js

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
please move to `metrics\server\lib\vis_data\helpers\`

Reply:
Moved